gsa/src/web/components/processmap/tools.js

```diff
@@ -54,11 +54,12 @@ const IconWrapper = styled.div`
   justify-content: space-around;
   padding: 10px 0 10px 0;
   & svg path {
-    fill: ${Theme.black};
+    fill: ${props => (props.inactive ? Theme.inputBorderGray : Theme.black)};
```

Comment:
Should be isInActive or even better use isActive

Reply:
It's active by default, so I chose it to be inactive that needs to be explicitly set. I'd go with isInactive, though.

Reply:
Plus, it's a different prop than the isActive used for DrawEdge und ConditionalColorization.